src/Uno.UI.Runtime.Skia.X11/X11PointerInputSource.cs

```diff
@@ -188,7 +188,7 @@ private PointerPoint CreatePointFromCurrentState(IntPtr time)
 		// This matches the format of WinUI. See also: https://github.com/unoplatform/uno/issues/14535
```

Comment:
This comment is now outdated I think?

Reply:
Hum didn't even noticed that comment, but actually no it's still relevant : I used ticks also here. We need to change that on all platforms, but this will be a much bigger change!

Reply:
I mean, the comment says that the code matches WinUI, but it's not as WinUI doesn't use ticks. Am I missing something?

Reply:
Updated issue to also fix this for X11